### DISCUSSION TITLE
Can stash open a folder containing the selected files?

### DISCUSSION THREAD

**Discussion by leonalux** [stashapp/stash#4014] [Q&A] [answered]
when i press 'Path:' in the 'file info', the folder not open.
Is there a way to open the folder that selected video in stash?

I'm copying the path and opening the folder.
I wonder if there is a more convenient way.

thx for made awesome app.

**Comment by DogmaDragon** [comment, marked as answer]
There isn't. As far as I know there are technical limitations that limit what different proccesses browser can access. 

There is an [userscript](https://docs.stashapp.cc/add-ons/userscripts/#stash-open-media-player) that allows to open fine in external media player, so maybe it can be modified to work with file manager.
